### ISSUE TITLE
Y-Axis Label Not Showing

### ISSUE THREAD

**Issue by MattyK14** [FormidableLabs/victory-native#97]
```
<View style={{ justifyContent: 'center', alignItems: 'center', flex: 1 }}>
      <Text style={{ color: '#fff', fontSize: 18 }}>Month Breakdown</Text>
      <VictoryChart
        domainPadding={5}
        animate={{ duration: 1000 }}
        width={height * 0.8}
        height={width * 0.8}
      >
      <VictoryAxis label="Month" />
      <VictoryAxis dependentAxis label="kWh" />
        <VictoryBar
          style={{ data: { fill: '#fff' } }}
          data={monthlyData}
          x="Month"
          y="Usage"
        />
      </VictoryChart>
    </View>
```

![simulator screen shot jun 2 2017 12 42 54 pm](https://cloud.githubusercontent.com/assets/16196171/26732138/1f162840-4791-11e7-970a-a0b2030c8475.png)

Could be related to https://github.com/FormidableLabs/victory/issues/281

**Comment by boygirl** [comment]
@MattyK14 Unfortunately this is a systemic issue with label rotation. It's a problem in the underlying `react-native-svg` lib. Issue here: https://github.com/react-native-community/react-native-svg/issues/242

**Comment by watadarkstar** [comment]
Following

**Comment by 18601673727** [comment]
Similar issue here, `angle` not working.

**Comment by boygirl** [comment]
Temporary fix in `victory-native@~0.16.0`

This fix uses a manual matrix transformation to rotate labels, but it is not compatible with other (user supplied) transformations. 

A better fix will be implemented pending the resolution of [this issue](https://github.com/react-native-community/react-native-svg/issues/242#issuecomment-344111728)

**Comment by boygirl** [comment]
Fixed
